x/cdp/genesis.go

```diff
@@ -80,7 +80,10 @@ func InitGenesis(ctx sdk.Context, k Keeper, pk types.PricefeedKeeper, sk types.S
 	k.SetNextCdpID(ctx, gs.StartingCdpID)
 	k.SetDebtDenom(ctx, gs.DebtDenom)
 	k.SetGovDenom(ctx, gs.GovDenom)
-	k.SetPreviousSavingsDistribution(ctx, gs.PreviousDistributionTime)
+	// only set the previous block time if it's different than default
```

Comment:
why is that?

Reply:
If the previous block time in genesis.json is 0, then we don't set it here (in InitGenesis) so that it can be set in the begin blocker to the time of the first block.

I guess if previous block time is 0 in genesis.json we could just set it the ctx.BlockTime in InitGenesis. But I'm not sure the difference between the first block time and genesis time.